code-gen-projects/java/code-gen-demo/src/test/java/org/example/CodeGenTest.java

```diff
@@ -26,7 +26,7 @@ class CodeGenTest {
          a.add("foo");
          a.add("bar");
          a.add("baz");
-         StructWithFields s = new StructWithFields("hello", 12, new AnonymousType2(a), 10e2);
+         StructWithFields s = new StructWithFields("hello", 12, new AnonymousType3(a), 10e2);
```

Comment:
This is exactly why we need #103

Reply:
Will be working on this in the next PR :)